apluslms_roman/cli.py

```diff
@@ -167,13 +168,16 @@ def create_parser(version=__version__,
         metavar=_('DIR'),
         default=[],
         action='append',
-        help=_("Change to directory DIR before reading the configuration file or doing anything else. If specified multiple times, then values are joined with `os.path.join`."))
+        help=_(
+            "Change to directory DIR before reading the configuration " +
```

Comment:
`+` is not required here. As a detail, '+' creates runtime code to calculate the addition, where as removing it tells compiler that here is a single string.

```
>>> pprint(parse("'foo' 'bar' 'baz'"))
Module(
    body=[
        Expr(
            lineno=1,
            col_offset=0,
            value=Str(lineno=1, col_offset=0, s='foobarbaz'),
        ),
    ],
)
>>> pprint(parse("'foo' + 'bar' + 'baz'"))
Module(
    body=[
        Expr(
            lineno=1,
            col_offset=0,
            value=BinOp(
                lineno=1,
                col_offset=14,
                left=BinOp(
                    lineno=1,
                    col_offset=0,
                    left=Str(lineno=1, col_offset=0, s='foo'),
                    op=Add(),
                    right=Str(lineno=1, col_offset=8, s='bar'),
                ),
                op=Add(),
                right=Str(lineno=1, col_offset=16, s='baz'),
            ),
        ),
    ],
)
```